### DISCUSSION TITLE
Career Readiness - Empty Week 4 Repository

### DISCUSSION THREAD

**Discussion by hoanglechau** [community/community#51254] [GitHub Education]
Hi everyone,

I joined the Career Readiness program of the Student Developer Pack, and got my 4 private repositories for the 4 weeks automatically created by GitHub. However, the repository for week 4 is completely empty. I've tried refreshing the page several times for a while, but currently, it's still empty. 

Could someone please help me with this issue? Thank you so much!

**Comment by samus-aran** [comment]
Hi @hoanglechau 👋 

Are you still experiencing this error?

**Reply by hoanglechau** [reply]
Hi @samus-aran 👋🏼  

Yes, my week 4 repository is still empty 😞 

Could you please help me with this issue? Thanks!

**Comment by artwalker** [comment]
Hi, do you solve that problem. My week 4 also doesn't have content.
